Changes

```diff
@@ -3,8 +3,16 @@ Working version
 
 ### Language features:
 
+<<<<<<< HEAD
 - #9331: Improve error messages for functor application and functor types.
   (Florian Angeletti and Gabriel Radanne, review by Leo White)
+=======
+- #10039: Safepoints
+  Add poll points to native generated code. These are effectively
+  zero-sized allocations and fix some signal and remembered set
+  issues. Also multicore prerequisite.
+  (Sadiq Jaffer, Anmol Sahoo, Stephen Dolan, reviewed by ??)
+>>>>>>> 3002287fb0... changes
```

Comment:
By the love of git, don't commit conflict markers!  

(I'm surprised that git even lets you do that in the first place.  I think Subversion does not.  It was a common mistake with CVS, though.)

Reply:
Oh dear. That must have happened during one of the rebases and been missed.